projects/shikshalokam/src/lib/core-module/components/headers/headers.component.ts

```diff
@@ -28,4 +29,8 @@ export class HeadersComponent implements OnInit {
     this.logoutClick.emit('logout');
   }
 
+  menuClick() {
+    this.sideNavClick.emit('');
```

Comment:
dont send it blank, send  "sideMenuToggle"

Reply:
changed